practice-one/src/App.jsx

```diff
@@ -5,13 +5,15 @@ import Navbar from './Components/Section/Navbar';
 import Header from './Components/Section/Header';
 import ListCard from './Components/Section/ProductList';
 import CtaSection from './Components/Section/CTA';
+import Footer from './Components/Section/Footer';
 
 const App = () => (
   <div>
```

Comment:
do we need this div

Reply:
it's not necessary, just use <></>.
Thanks for reminding